src/Docker.DotNet/Endpoints/IContainerOperations.cs

```diff
@@ -95,11 +95,23 @@ public interface IContainerOperations
         /// 500 - Server error.
         /// </remarks>
         /// <param name="id">ID or name of the container.</param>
-        [Obsolete("Use 'Task GetContainerLogsAsync(string id, ContainerLogsParameters parameters, CancellationToken cancellationToken, IProgress<string> progress)'")]
+        [Obsolete("The stream returned by this method won't be demultiplexed properly if the container was created without a TTY. Use GetContainerLogsAsync(string, bool, ContainerLogsParameters, CancellationToken) instead")]
         Task<Stream> GetContainerLogsAsync(string id, ContainerLogsParameters parameters, CancellationToken cancellationToken = default(CancellationToken));
 
+        [Obsolete("The strings reported by this method will contain junk if the container was created without a TTY. Use GetContainerLogsAsync(string, bool, ContainerLogsParameters, CancellationToken) instead")]
```

Comment:
I dont think we should obsolete this. Just add a comment that you want to use this only if `tty=false` to use with `tty=true` instead use the one you added.

Reply:
Done